apps/lfx-one/src/styles.scss

```
@@ -11,7 +11,7 @@
 
   /* Typography Defaults - Inter on html so it cascades to everything */
   html {
-    font-family: var(--font-inter);
+    font-family: 'Inter', ui-sans-serif, system-ui, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)